tox.ini

```diff
@@ -1,12 +1,12 @@
 # For more information about tox, see https://tox.readthedocs.io/en/latest/
 [tox]
-envlist = py{37,38,39}-{,coredev}
+envlist = py{37,38,39}{-corerelease,-coredev}
```

Comment:
I'm not familiar with this syntax. `coredev` seems to defined below as the GH repo. Is `corerelease` a standard thing, or does it need to be defined?